MobileSyncExplorerSwift/MobileSyncExplorerSwift/ViewControllers/ContactDetailViewController.swift

```diff
@@ -226,7 +226,7 @@ class ContactDetailViewController: UniversalViewController {
                                                     deleteButton.heightAnchor.constraint(equalToConstant: 44.0),
                                                     deleteButton.bottomAnchor.constraint(equalTo: scrollView.bottomAnchor, constant: -compInset)
             ])
-    
+        super.applyConstraints()
```

Comment:
In iO3 13 `traitCollectionDidChange` isn't called at this point anymore (sounds like the size is estimated instead) which we were using for the initial constraints so applying the constraints here too